test/e2e/main_test.go

```diff
@@ -114,17 +114,12 @@ func TestTargetsUp(t *testing.T) {
 
 	targets := []string{
 		"node-exporter",
-		"kubelet",
-		//"scheduler",
-		//"kube-controller-manager",
-		"apiserver",
 		"kube-state-metrics",
 		"prometheus-k8s",
 		"prometheus-operator",
 		"alertmanager-main",
 		"crio",
```

Comment:
What about `crio`? It is also tested in origin tests (aws-e2e): https://github.com/openshift/origin/blob/master/test/extended/prometheus/prometheus.go#L146

Reply:
let's see what CI says, I'm happy to do a follow up, but I'd like to unblock us rather sooner than do another run if e2e-aws agrees :)

Reply:
nevermind, CI failed, so I removed this as well :) good catch!